tools/streamer_recorder/README.md

```diff
@@ -0,0 +1,74 @@
+# libfreenect2 Streamer/Recorder toolbox
+
+## Table of Contents
+
+* [Description](README.md#description)
+* [Maintainers](README.md#maintainers)
+* [Installation](README.md#installation)
+  * [Windows / Visual Studio](README.md#windows--visual-studio)
+  * [MacOS X](README.md#mac-osx)
+  * [Linux](README.md#linux)
+
+
+## Description
+
+Additional toolbox featuring:
+- UDP streaming of kinect captured images (``-streamer`` option)
+- Recording of kinect captured images to disk (``-recorder`` option)
+
+## Maintainers
+
+* David Poirier-Quinot
+
+## Installation
+
+### Windows / Visual Studio
+
+### Mac OSX
+
+* Install OPENCV
+
+    ```
+brew install opencv3
+```
+
+* Install Numpy for Blender viewer
+
+    ```
+pip3 install numpy
+```
+
+and link numpy and cv2 to blender python3 site-package (rename / remove old numpy if needed)
+
+(tested with 1.10.4, previous versions happened to raise ``ImportError: numpy.core.multiarray failed to import`` when typing ``import cv2`` in python)
+
+* Build
+
+    ```
+mkdir build && cd build
+cmake ..
+make
+make install
+```
+* Run the test program: `.ProtonectSR`
```

Comment:
Can you use some more direct name than `ProtonectSR`? `kinect2-stream` `kinect2-record`?

I expect you split it to two tools, because DOTADIW.

Reply:
@xlz -- in my updates I am making I can rename the executables if you like to say what would be better called `freenect2-record` and `freenect2-stream`, but these would come from the same code base with the presumption of options `-streamer` or `-recorder` based on the executable chosen. However, it is quite plausible to use either one or the other or both, if you split them, how would you recombined them if you need to both stream and record at the same time?

The more I look at it the more I see it as an extension of the Protonect functionality where you want all available pipelines, pausing, streaming, and recording optionally at any time. Thoughts.

Reply:
Added executables for reference `freenect2-record`, `freenect2-stream`, `freenect2-replay`. Won't split as need the rest of Protonect's code.